### PR TITLE
Proxy Cache plugins: add extra warning about exact content type matches

### PR DESCRIPTION
**Pull request by tlblessing**
https://konghq.atlassian.net/browse/DOCS-335

Added extra warning underneath params table. 

Direct review links (see the warning underneath the parameters table):
- https://deploy-preview-2402--kongdocs.netlify.app/hub/kong-inc/proxy-cache/#parameters
- https://deploy-preview-2402--kongdocs.netlify.app/hub/kong-inc/proxy-cache-advanced/#parameters

### CODE REVIEW

File: app/_hub/kong-inc/proxy-cache-advanced/index.md [hunk 1]
```diff
@@ -152,6 +152,12 @@ params:
       value_in_examples:
       description: |
         Unhandled errors while trying to retrieve a cache entry (such as redis down) are resolved with `Bypass`, with the request going upstream.
+  extra: |
+
+    **Warning:** The `content_type` requires an exact match. For example, if your Upstream expects
+    `application/json; charset=utf-8` and the `config.content_type` value is only `application/json`
+    (a partial match), then the proxy cache is bypassed.
```
Comment (lena-larionova) [inline app/_hub/kong-inc/proxy-cache-advanced/index.md:159]:
```suggestion
    <div class="alert alert-ee red">
    <strong>Warning:</strong> The <code>content_type</code> parameter requires
    an exact match. For example, if your Upstream expects
    <code>application/json; charset=utf-8</code> and the
    <code>config.content_type</code> value is only <code>application/json</code>
    (a partial match), then the proxy cache is bypassed.
    </div>
```
You can use html in these pipe extended descriptions with no problems.

File: app/_hub/kong-inc/proxy-cache/index.md [hunk 2]
```diff
@@ -97,6 +97,11 @@ params:
       value_in_examples:
       description: |
         The name of the shared dictionary in which to hold cache entities when the memory strategy is selected. Note that this dictionary currently must be defined manually in the Kong Nginx template.
+  extra: |
+
+    **Warning:** The `content_type` requires an exact match. For example, if your Upstream expects
+    `application/json; charset=utf-8` and the `config.content_type` value is only `application/json`
+    (a partial match), then the proxy cache is bypassed.
```
Comment (lena-larionova) [inline app/_hub/kong-inc/proxy-cache/index.md:104]:
```suggestion
    <div class="alert alert-ee red">
    <strong>Warning:</strong> The <code>content_type</code> parameter requires
    an exact match. For example, if your Upstream expects
    <code>application/json; charset=utf-8</code> and the
    <code>config.content_type</code> value is only <code>application/json</code>
    (a partial match), then the proxy cache is bypassed.
    </div>
```

File: app/_hub/kong-inc/proxy-cache-advanced/index.md (hunk 1 above)
Comment (tlblessing) [reply app/_hub/kong-inc/proxy-cache-advanced/index.md:159]: > You can use html in these pipe extended descriptions with no problems.

I was having problems with it:  `[16:48:32]              Error: YAML Exception reading /Users/terryblessing/github/docs.konghq.com/app/_hub/kong-inc/proxy-cache/index.md: (<unknown>): could not find expected ':' while scanning a simple key at line 102 column 1`

I'll try committing your suggestions.

Comment (tlblessing) [reply app/_hub/kong-inc/proxy-cache-advanced/index.md]: @lena-larionova it worked but with the highlighting changes underway, it looks a bit odd. Can we use red text within these warnings only?

image

Comment (lena-larionova) [reply app/_hub/kong-inc/proxy-cache-advanced/index.md]: Sure, I can add it to the alerts overhaul ticket: https://konghq.atlassian.net/browse/DOCS-1379

**Review by lena-larionova — APPROVED** [review]
LGTM, and I've added the concern about code in alert blocks to the docs ticket for alert overhaul (https://konghq.atlassian.net/browse/DOCS-1379).

**Comment by tlblessing** [comment]
> LGTM, and I've added the concern about code in alert blocks to the docs ticket for alert overhaul (https://konghq.atlassian.net/browse/DOCS-1379).

Thanks @lena-larionova !

**Pull request merged**